src/access/universal.test.ts

```diff
@@ -1268,3 +1271,113 @@ describe("getGroupAccessAll", () => {
     expect(access).toBeNull();
   });
 });
+
+describe("getAccessFor", () => {
+  it("calls to getAgentAccess with the appropriate parameters", async () => {
+    const universalModule = jest.requireActual("./universal") as {
+      getAgentAccess: () => Promise<Access | null>;
+    };
+    universalModule.getAgentAccess = jest.fn();
+    const options = {
+      fetch: jest.fn().mockResolvedValue({
+        ok: false,
+      } as never) as typeof fetch,
+    };
+    await getAccessFor(
+      "https://some.resource",
+      "agent",
+      "https://some.pod/profile#webid",
+      options
+    );
+    expect(universalModule.getAgentAccess).toHaveBeenCalledWith(
+      "https://some.resource",
+      "https://some.pod/profile#webid",
+      expect.objectContaining(options)
+    );
+  });
+
+  it("throws if the agent has been omited", async () => {
```

Comment:
A double T, too :smile:  (Likewise for the other one.)

```suggestion
  it("throws if the agent has been omitted", async () => {
```

Reply:
Next time, I'll write "is missing" :p.

I had a French teacher who always said "If you don't know how to write bicycle, just write bike", and I felle I should have taken this piece of advice more seriously.

Reply:
Your French teacher was very wise :smile: